src/Workspaces/CSharp/Portable/OrganizeImports/CSharpOrganizeImportsService.Rewriter.cs

```diff
@@ -20,8 +17,6 @@ private sealed class Rewriter(OrganizeImportsOptions options) : CSharpSyntaxRewr
         private readonly bool _separateGroups = options.SeparateImportDirectiveGroups;
         private readonly SyntaxTrivia _fallbackTrivia = CSharpSyntaxGeneratorInternal.Instance.EndOfLine(options.NewLine);
 
-        public readonly IList<TextChange> TextChanges = [];
```

Comment:
Any idea what this had been intended to be used for?

Reply:
not a clue :D